paimon-spark/paimon-spark-common/src/main/scala/org/apache/paimon/spark/SparkSource.scala

```diff
@@ -80,9 +82,15 @@ class SparkSource
   }
 
   private def loadTable(options: JMap[String, String]): DataTable = {
+    val path = CoreOptions.path(options)
     val catalogContext = CatalogContext.create(
-      Options.fromMap(mergeSQLConf(options)),
-      SparkSession.active.sessionState.newHadoopConf())
+      Options.fromMap(
```

Comment:
For SparkSource loadTable, maybe we can just keep the original way, because it is simpler and more general through `spark.read.format("paimon").options()` to set table level config. And it is difficult to get catalogname and dbname here